Cargo.toml

```diff
@@ -107,3 +107,6 @@ backend-s3 = ["nydus-storage/backend-s3"]
 
 [workspace]
 members = ["api", "builder", "clib", "rafs", "storage", "service", "utils"]
+
+[patch.crates-io]
+tar = { git = "https://github.com/nydusaccelerator/tar-rs.git" }
```

Comment:
Maybe add a comment here explaining why we need to use the forked code.

Reply:
OK, comment added.